src/plugins/data/common/search/aggs/buckets/_terms_other_bucket_helper.ts

```diff
@@ -209,7 +209,7 @@ export const buildOtherBucketAgg = (
       exhaustiveBuckets = false;
     }
     if (aggIndex < index) {
-      each(agg.buckets, (bucket: any, bucketObjKey) => {
+      each(agg?.buckets, (bucket: any, bucketObjKey) => {
```

Comment:
Maybe the line above should be updated to handle empty buckets too?

Something like `if ((aggregations[aggId]?.buckets.length ?? 0) < 1) {`

Reply:
Agree on the aim of the suggestion.
Maybe I would make it just a little bit more verbose:

```ts
const agg = aggregations[aggId];
if (!agg || agg.buckets.length) {
  noAggBucketResults = true;
  return;
}
...
```

Reply:
Done https://github.com/elastic/kibana/pull/150406/commits/6123416f61c44e9e3fed78fe76e6e629a0ccd39d